### PR TITLE
docs: cleaning up some api generated docs

### PR DESCRIPTION
**Pull request by msdolbey**
`\n` is in some generated docs. This is short
term fix until we can make the generation of said
api docs better.

## Jira Ticket
n/a

## Description of changes being made
`\n` is ugly to see in docs. This is coming from docs that are automatically generated from API definitions. This will need to get changed in the Konvoy code. I have an idea how to do that, but this is quicker and will be able to get done quickly for now. I'll open a subsequent ticket for this 

## Checklist
- [ ] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [ ] Test all commands and procedures, if applicable.
- [ ] Create your PR against `main`. 
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://wiki.d2iq.com/display/ENG/Contributing+to+Docs) for more information.

### CODE REVIEW

File: pages/dkp/konvoy/1.8/access-authentication/api-reference/index.md [hunk 1]
```diff
@@ -68,9 +68,9 @@ ConnectorSpec defines the desired state of Connector
...
 | loadAllGroups | Flag which indicates that all user groups and teams should be loaded. | *bool | false |
-| teamNameField | Optional choice between 'name' (default), 'slug', or 'both'.\n\nAs an example, group claims for member of 'Site Reliability Engineers' in Acme organization would yield:\n  - ['acme:Site Reliability Engineers'] for 'name'\n  - ['acme:site-reliability-engineers'] for 'slug'\n  - ['acme:Site Reliability Engineers',\n     'acme:site-reliability-engineers'] for 'both' | *string | false |
+| teamNameField | Optional choice between 'name' (default), 'slug', or 'both'.As an example, group claims for member of 'Site Reliability Engineers' in Acme organization would yield:  - ['acme:Site Reliability Engineers'] for 'name'  - ['acme:site-reliability-engineers'] for 'slug'  - ['acme:Site Reliability Engineers',     'acme:site-reliability-engineers'] for 'both' | *string | false |
```
Comment (msdolbey) [inline pages/dkp/konvoy/1.8/access-authentication/api-reference/index.md:73]:
```suggestion
| teamNameField | Optional choice between 'name' (default), 'slug', or 'both'. As an example, group claims for member of 'Site Reliability Engineers' in Acme organization would yield:  - ['acme:Site Reliability Engineers'] for 'name'  - ['acme:site-reliability-engineers'] for 'slug'  - ['acme:Site Reliability Engineers',     'acme:site-reliability-engineers'] for 'both' | *string | false |
```

File: pages/dkp/konvoy/1.7/access-authentication/api-reference/index.md [hunk 2]
```diff
@@ -68,9 +68,9 @@ ConnectorSpec defines the desired state of Connector
...
 | loadAllGroups | Flag which indicates that all user groups and teams should be loaded. | *bool | false |
-| teamNameField | Optional choice between 'name' (default), 'slug', or 'both'.\n\nAs an example, group claims for member of 'Site Reliability Engineers' in Acme organization would yield:\n  - ['acme:Site Reliability Engineers'] for 'name'\n  - ['acme:site-reliability-engineers'] for 'slug'\n  - ['acme:Site Reliability Engineers',\n     'acme:site-reliability-engineers'] for 'both' | *string | false |
+| teamNameField | Optional choice between 'name' (default), 'slug', or 'both'.As an example, group claims for member of 'Site Reliability Engineers' in Acme organization would yield:  - ['acme:Site Reliability Engineers'] for 'name'  - ['acme:site-reliability-engineers'] for 'slug'  - ['acme:Site Reliability Engineers',     'acme:site-reliability-engineers'] for 'both' | *string | false |
```
Comment (msdolbey) [inline pages/dkp/konvoy/1.7/access-authentication/api-reference/index.md:73]:
```suggestion
| teamNameField | Optional choice between 'name' (default), 'slug', or 'both'. As an example, group claims for member of 'Site Reliability Engineers' in Acme organization would yield:  - ['acme:Site Reliability Engineers'] for 'name'  - ['acme:site-reliability-engineers'] for 'slug'  - ['acme:Site Reliability Engineers',     'acme:site-reliability-engineers'] for 'both' | *string | false |
```

**Review by cneth — APPROVED** [review]

**Review by samvantran — COMMENTED** [review]
Thanks for fixing but won't this get overwritten with any of the sync PRs?

**Comment by msdolbey** [comment]
> Thanks for fixing but won't this get overwritten with any of the sync PRs?

Indeed it will - there's that related d2iq JIRA ticket to resolve those issues. Otherwise we'd have to edit those sync PRs overtime. It won't mess up this week's though.

**Review by gracedo — APPROVED** [review]

**Pull request merged**